package.json

```diff
@@ -4,14 +4,31 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "webpack-dev-server --content-base ./"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/nomad302/demo-app-ts-react.git"
   },
   "author": "\"sagar\"",
   "license": "ISC",
+  "devDependencies": {
+    "@types/react": "^16.9.36",
+    "@types/react-dom": "^16.9.8",
+    "awesome-typescript-loader": "^5.2.1",
+    "contentful": "^7.14.4",
```

Comment:
You can move this to dependencies. It is not a development dependency.

Reply:
only this dependency "contentful": "^7.14.4", right?

Reply:
Yes

Reply:
Thanks!!